docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -111,10 +111,34 @@ Preparing For a Release
       # Use system site packages e.g. so that "rpm" can be imported
       $ mkvirtualenv -a "flocker-${VERSION}" --system-site-packages "flocker-${VERSION}"
       (flocker-0.1.2)$ pip install --ignore-installed --editable .[dev]
-      (flocker-0.1.2)$ admin/create-release-branch --flocker-version=${VERSION}
+
+   Create or switch to the correct branch. If this is a major or minor release, create a new branch
+
+   .. prompt:: bash $ auto
+
+      $ git checkout -b release/flocker-1.2 origin/master
+
+   If this is a patch release, checkout the release branch
+
+   .. prompt:: bash $ auto
+
+      $ git checkout -b release/flocker-1.2 origin/master
+
+   If this is a development release, create a new branch
+
+   .. prompt:: bash $ auto
+
+      $ git checkout -b release/flocker-1.2.0.dev1 origin/master
+
+#. Ensure the copyright is up-to-date.
+
+   .. prompt:: bash $,(flocker-0.1.2)$ auto
+
```

Comment:
This should be automated, but I decided to describe it as part of the release process for the first pass of design review.